crates/bevy_winit/src/state.rs

```diff
@@ -525,10 +558,11 @@ impl<T: Event> ApplicationHandler<T> for WinitAppRunnerState<T> {
                 self.redraw_requested = true;
             }
 
+            self.update_cursors(event_loop);
```

Comment:
very minor, but its not obvious to me why this should be here rather than at the top where we sync windows? i guess it makes sense not to try for new cursors if we're not about to render?

Reply:
I don't know what you by "where we sync windows".

`self.app_update()` a couple of lines above runs the world schedule that causes cursors to be changed, so it makes sense to apply the updates as soon as possible. I guess I could move this line to just after `self.run_app_update()` inside the if statement.

Reply:
:+1: I moved it there to avoid all unnecessary updates